src/components/NftDetailsPanel.vue

```diff
@@ -104,6 +104,7 @@
 </template>
 
 <script setup>
+
```

Comment:
Most of our files don't use that additional newline after `<script setup>` Is there a reason for this sudden change?

Reply:
oops, fixed